src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -157,11 +159,49 @@ class DepositServlet(app: EasyDepositApiApp)
       ).getOrRecoverResponse(respond)
   }
 
-  private def postFileItem(uuid: UUID, path: Path, item: FileItem): Try[(Boolean, FileItem)] = {
-    val fullPath = path.resolve(item.name)
-    managed(item.getInputStream)
-      .apply(app.writeDepositFile(_, user.id, uuid, fullPath))
-      .map((_, item))
+  private def uploadFileItem(uuid: UUID, path: Path, uploadItem: FileItem): Try[(Boolean, FileItem)] = {
+    val extensionIsZip = uploadItem.name.matches(".*.g?z(ip)?")
+    lazy val contentTypeIsZip = uploadItem.contentType.exists(_.matches(
+      "(application|multipart)/(x-)?g?zip(-compress(ed)?)?( .*)?"
+    ))
+    logger.debug(s"is it a ZIP? ${ uploadItem.name } : $extensionIsZip; ${ uploadItem.contentType } : $contentTypeIsZip ")
```

Comment:
a question in the debug log? that doesn't really make much sense, if you ask me.
Can't we have something here like `"it is a ZIP : <etc.>"` or `"it is a XXX : <etc.>"`, depending on the specific upload.